Core/Core/Conversations/GetConversations.swift

```diff
@@ -85,3 +102,22 @@ public class GetConversation: APIUseCase {
         self.include = include
     }
 }
+
+public class UpdateConversation: APIUseCase {
+    public var cacheKey: String?
+    public typealias Model = Conversation
+    public let id: String
+    public let state: ConversationWorkflowState
+
+    public var request: PutConversationRequest {
+        return PutConversationRequest(id: id, workflowState: state)
+    }
+
+    public var scope: Scope { Scope(predicate: NSPredicate(format: "%K == %@", "id", id), order: []) }
+
+    public init(id: String, state: ConversationWorkflowState) {
+        self.id = id
+        self.state = state
+    }
+
+}
```

Comment:
I would expect this use case to save the updated coredata model. Is that happening automatically?

Reply:
It is saving.  Although when this request returns, it's missing a field we display in the cell, so look at this commit https://github.com/instructure/canvas-ios/pull/1278/commits/07f6b9c64c90abca97246e882462a56b35499166 . Let me know your thoughts.

Reply:
I think that's a good way to go about it. If we know the response to the update can't be saved as a whole, only getting the parts we expect to have changed is great.